bin/validate_bigquery

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+import click
+from pathlib import Path
+from git import Repo
+from shutil import copyfile
+import json
+import difflib
+import sys
+
+BASE_DIR = Path("/app").resolve()
+
+
+@click.group()
+def validate():
+    pass
```

Comment:
Was this be design? The default behavior of this script is to do nothing?

Reply:
This is just the group entrypoint for click.